server/src/lib/model.ts

```diff
@@ -23,10 +24,14 @@ export default class Model {
     this.config = config;
     this.db = new DB(this.config);
     this.users = new Users();
-    this.clips = new Clips();
+    this.clips = new Clips(this.db);
     this.loaded = false;
   }
 
+  set isMigrated(val: boolean) {
```

Comment:
why are we using a setter? seems like a private member would do the same thing, and follow the convention currently set in the code

Reply:
the Model itself doesn't need the flag, so the setter is just used to forward the flag